### PR TITLE
Traceback tag: Emphasize reason for sharing traceback

### PR DESCRIPTION
**Pull request by swfarnsworth**
The previous version talks about how to read a traceback, but this tag is most often used to help the asker provide the traceback. This version is more brief and emphasizes why they are being asked to provide the traceback, and uses an example that I think is slightly more illustrative.

### CODE REVIEW

**Comment by swfarnsworth** [comment]
I meant to include a link to the pastebin for long tracebacks. Please don't merge this until I add that.

File: bot/resources/tags/traceback.md [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+Please avoid screenshots so we can copy and paste parts of the message.
 
 A full traceback could look like:
 ```py
 Traceback (most recent call last):
-    File "tiny", line 3, in
-        do_something()
+    File "my_file.py", line 3, in
+        add_three("6")
     File "tiny", line 2, in do_something
```
Comment (ToxicKidz) [inline bot/resources/tags/traceback.md:11]:
```suggestion
    File "my_file.py", line 2, in add_three
```

**Review by ToxicKidz — CHANGES_REQUESTED** [review]
Looks good, just one comment regarding the traceback.

File: bot/resources/tags/traceback.md (hunk 1 above)
Comment (swfarnsworth) [reply bot/resources/tags/traceback.md]: Thank you for pointing this out; I have fixed it.

**Review by ToxicKidz — APPROVED** [review]
LGTM :shipit:

**Comment by swfarnsworth** [comment]
I think we can do a squash merge for this one?

**Review by Xithrius — APPROVED** [review]
Sorry I took so long to review. After reading over it multiple times, I see no error in your ways, and I have no suggestions to improve. Off it goes, to production.

**Pull request merged**